content/collections/fieldtypes/bard.md

```diff
@@ -127,11 +127,11 @@ An alternative approach (for those who like DRY or small templates) is to create
 ```language-files
 partials/
 |-- sets/
-|   |-- gallery.html
-|   |-- image.html
-|   |-- poll.html
-|   |-- text.html
-|   |-- video.html
+|   |-- _gallery.antlers.html
+|   |-- _image.antlers.html
+|   |-- _poll.antlers.html
+|   |-- _text.antlers.html
+|   |-- _video.antlers.html
```

Comment:
We'll want `.antlers`, (although just `.html` should technically work too, it just wouldn't be antlers), but the underscore shouldn't matter.

Can you remove the underscores?

If it's really not working for you without underscores, can you open an issue in the cms repo instead?

Reply:
Done!